File: solutions/alfi.compitest/source_gen/alfi/compitest/ActivityWhileStatementTest.cpp
#include "ActivityWhileStatementTest.h"
#include <alf/library/BasicInputOutput.hpp>
#include <string>

void ActivityWhileStatementTest::WhileStatementTest()
{
  int a = 0;
  while (a < 10)
  {
    a = a + 1;
    alf::library::BasicInputOutput::WriteLine(std::to_string(a));
  }
  alf::library::BasicInputOutput::WriteLine(std::string("End: ") + std::to_string(a));
}

File: solutions/alfi.compitest/source_gen/alfi/compitest/ActivityWhileStatementTest.cs
/// <filename>
///     ActivityWhileStatementTest.cs
/// </filename>
using Alf.Library;
using System;

public class ActivityWhileStatementTest
{
    public static void Execute()
    {
        int a = 0;
        while (a < 10)
        {
            a = a + 1;
            BasicInputOutput.WriteLine(Convert.ToString(a));
        }
        BasicInputOutput.WriteLine("End: " + Convert.ToString(a));
    }

    public static void Main(string[] args)
    {
        ActivityWhileStatementTest.Execute();
    }
}
